hibike/devices/YogiBear/LED.cpp

```diff
@@ -28,6 +31,9 @@ void ctrl_YELLOW() {
 	} else {
 		digitalWrite(LED_YELLOW, LOW);
 	}
+	if(!isMotorEnabled()) {
```

Comment:
See above, found the extra function calls somewhat unnecessary for this simple feature